changelogs/fragments/73114-fix-ansible-role-download.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - galaxy - role install does not download artifact from API response `download_url` location (https://github.com/ansible/ansible/issues/73103).
```

Comment:
I'm pretty sure it's a feature, not a bug.

Reply:
Updated...thanks!